modules/lambdas/main.tf

```diff
@@ -479,3 +479,300 @@ data "aws_lambda_invocation" "db_migration" {
 }
 
 ## TODO: Should create null resource to handle password changes ORCA-145
+
+# API Gateway- API for orca_catalog_reporting lambda function
+resource "aws_api_gateway_rest_api" "orca_catalog_reporting_api" {
+  name        = "${var.prefix}_orca_catalog_reporting_api"
+  description = "API for catalog reporting lambda function"
+  endpoint_configuration {
+    types = ["PRIVATE"]
+    # Cumulus might need to create vpc_endpoint_ids which should be added here in the future in order to access this API
+  }
+}
+resource "aws_api_gateway_resource" "orca_catalog_reporting_api_resource" {
+  path_part   = "{proxy+}"
+  parent_id   = aws_api_gateway_rest_api.orca_catalog_reporting_api.root_resource_id
+  rest_api_id = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+}
+
+resource "aws_api_gateway_method" "orca_catalog_reporting_api_method" {
+  rest_api_id = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+  resource_id = aws_api_gateway_resource.orca_catalog_reporting_api_resource.id
+  http_method = "ANY"
+  # todo: Make sure this is locked down against external access.
+  authorization = "NONE"
+}
+
+resource "aws_api_gateway_integration" "orca_catalog_reporting_api_integration" {
+  rest_api_id             = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+  resource_id             = aws_api_gateway_resource.orca_catalog_reporting_api_resource.id
+  http_method             = aws_api_gateway_method.orca_catalog_reporting_api_method.http_method
+  integration_http_method = "POST"
+  type                    = "AWS"
+  uri                     = aws_lambda_function.orca_catalog_reporting.invoke_arn
+}
+
+resource "aws_api_gateway_method_response" "orca_catalog_reporting_response_200" {
+  rest_api_id = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+  resource_id = aws_api_gateway_resource.orca_catalog_reporting_api_resource.id
+  http_method = aws_api_gateway_method.orca_catalog_reporting_api_method.http_method
+  status_code = "200"
+}
+
+resource "aws_api_gateway_integration_response" "orca_catalog_reporting_api_response" {
+  depends_on  = [aws_api_gateway_integration.orca_catalog_reporting_api_integration]
+  rest_api_id = aws_api_gateway_rest_api.orca_catalog_reporting_api.id
+  resource_id = aws_api_gateway_resource.orca_catalog_reporting_api_resource.id
+  http_method = aws_api_gateway_method.orca_catalog_reporting_api_method.http_method
+  status_code = aws_api_gateway_method_response.orca_catalog_reporting_response_200.status_code
+
+  # Transforms the backend JSON response to XML
+  response_templates = {
+    "application/json" = <<EOF
+    #set($inputRoot = $input.path('$'))
```

Comment:
it was there to transform backend JSON response to XML. I don't think it is being used but kept it since it was in this PR 
https://github.com/nasa/cumulus-orca/pull/102

Reply:
I would remove it not being used.

Reply:
Never mind, looking into templates, I see now that is how code logic is added